### PR TITLE
Creation of MOM of Community Meeting held on 28/01/2023

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
Signed off by Bhaswati Roy

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: working-groups/community_meeting/28-01-2023.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+[@abhilipsasahoo03](https://github.com/abhilipsasahoo03)
+
+[@jigarvyasidea](http://github.com/jigarvyasidea)
+
+[@ayush78781](https://github.com/ayush78781)
+
+## Agenda
+
+Disscusion on ideas and issues worked on by the community members.
```
Comment (samyakshah3008) [inline working-groups/community_meeting/28-01-2023.md:23]: Can you please correct typo error from `Disscusion` to `Discussion`?

File: working-groups/community_meeting/28-01-2023.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Sahil talked about the errors he was facing during the local installation of [GCBM.Belize](https://github.com/moja-global/GCBM.Belize) via windows method. Namya and Karan recommended installing via the Docker method which is the most preferred method.
+- Abhilipsa updated us about the new blog she wrote on Grace Hopper Celebration Open Source Day for the community website. Being a mentor representing Moja Global, she precisely presented her experience.
+- Aman joined his first community meeting and informed us that he was new to Moja Global. Namya gave him an overview of Moja global and its projects. She also recommended him to check [Moja Global Documentation](https://docs.moja.global/) and [FLINT Handbook](https://moja-global.github.io/Handbook/)
+- Bhaswati notified us about her new idea of introducing a project structure to the README of the FLINT handbook and community website
+- Jigar discussed about how he could get started with the Handbook, create issues, and work on them. He also put up a discussion on writing blogs for Moja Global
```
Comment (samyakshah3008) [inline working-groups/community_meeting/28-01-2023.md:32]: Can you please end both the lines with full stop?

**Review by samyakshah3008 — APPROVED** [review]
PR looks good, just added minor changes which you can work on. 
Rest is all fine.  Great work Bhaswati!

**Comment by BhaswatiRoy** [comment]
Hello @samyakshah3008
Thanks for the review, I have made the necessary changes

**Comment by sailorworks** [comment]
@Namyalg

**Pull request merged**